helmfile.d/20-dashboard.yaml

```diff
@@ -61,6 +61,38 @@ releases:
       - name: "grafana\\.ini.auth\\.generic_oauth.token_url"
         value: "https://{{ .Values.server_name }}/managementportal/oauth/token"
 
+  - name: data-dashboard-backend
+    chart: radar/data-dashboard-backend
+    version: {{ .Values.data_dashboard_backend._chart_version }}
+    installed: {{ .Values.data_dashboard_backend._install }}
+    values:
+      - "../etc/data-dashboard-backend/values.yaml"
+      - {{ .Values.data_dashboard_backend | toYaml | indent 8 | trim }}
+    set:
+      - name: serverName
+        value: {{ .Values.server_name }}
+      - name: ingress.hosts
+        values:
+          - data.{{ .Values.server_name }}
+      - name: jdbc.user
+        value: {{ .Values.timescaledb_username }}
+      - name: jdbc.password
+        value: {{ .Values.timescaledb_password }}
+      - name: jdbc.url
+        value: {{ dig "jdbc" "url" (printf "jdbc:postgresql://timescaledb-postgresql-hl:5432/%s" .Values.timescaledb_db_name) .Values.data_dashboard_backend }}
+      - name: managementPortal.clientSecret
+        value: {{ .Values.management_portal.oauth_clients.radar_data_dashboard_backend.client_secret }}
+
+  - name: kafka-data-transformer
```

Comment:
It is better that the the chart name and the app name would be the same so it would makes finding what has been installed easier.

Reply:
> How is this different than ksql_server that is being installed here?
In principle it is identical. How could we make sure that we define this once?

Reply:
What is stopping us from defining this once?

Reply:
The service is shared between _20-dashboard.yaml_ and _40-realtime-dashboard.yaml_. How would this work?

Reply:
Typically applications can accept multiple users and connections similar to Kafka and Postgresql. I suspect if both dashboard and realtime dashboard use once KSQL instance at the same time they wouldn't interfere with each other.  But I'm not sure about it.
Also maybe we can merge 20-dashboard.yaml and 40-realtime-dashboard.yaml into fine dashboards.yaml file

Reply:
> Also maybe we can merge 20-dashboard.yaml and 40-realtime-dashboard.yaml into fine dashboards.yaml file

Yeah, that's what I see as a 'problem'. When using a single KSQLDB definition, these files must be merged, right?

Reply:
I have merged the ` 40-realtime-analysis.yaml` file into the `20-dashboard.yaml` file.